packages/rest/src/controllers/connections/ConnectionController.ts

```diff
@@ -48,6 +51,26 @@ export class ConnectionController {
     return connections.map((c) => c.toJSON())
   }
 
+  /**
+   * Send a basic message to a connection
+   */
+  @Post('/:connectionId/send-message')
+  @OnUndefined(204)
+  public async sendMessage(
```

Comment:
As this is part of a separate module, do you think it should be in a different controller with different path (e.g. `/basic-messages/`)